src/datasets/non_desktop/google_play_store/reference.md

```diff
@@ -0,0 +1,115 @@
+
+There are other tables available that are yet to be explored / incorporated into regular reporting. Some that may be of interest include: (Crashes table by app_version, device and os; installs tables and ratings tables).
+
+The metrics included in the retained_installers tables are:
+- **Store_Listing_Visitors** - Unique users who visited the app’s store listing on the Play Store app but haven’t installed the app
+- **Installers** - Unique users who installed the app after visiting the app’s store listing on the Play Store app. 
+- **Visitors_to_installer_conversion_rate** - Percentage of Store_Listing_visitors that install the app.
+- **Installers_retained_for_1_day** - Installers who kept the app on at least one of their devices for 1 day. Installation doesn’t mean the app was opened over this period. 
+- **Installers_to_1_day_retention_rate** - Percentage of installers who have the app on one of their devices 1 day after install.
+- **Installers_retained_for_7_days** - Installers who kept the app on at least one of their devices for 7 days. Installation doesn’t mean the app was opened over this period. 
+- **Installers_to_7_days_retention_rate** - Percentage of installers who have the app on one of their devices 7 days after install.
+- **Installers_retained_for_15_days** - Installers who kept the app on at least one of their devices for 15 days. Installation doesn’t mean the app was opened over this period. 
+- **Installers_to_15_days_retention_rate** - Percentage of installers who have the app on one of their devices 15 days after install.
+- **Installers_retained_for_30_days** - Installers who kept the app on at least one of their devices for 30 days. Installation doesn’t mean the app was opened over this period. 
+- **Installers_to_30_days_retention_rate** - Percentage of installers who have the app on one of their devices 30 days after install.
+
+## Background and Caveats
+
+As of Aug 25 2020 not all the tables available in the dataset have been explored and vetted for accuracy by the data science team. Tables that were fully reviewed and being documented here are the Retained_installers tables whose primary use case is to explain acquisition. 
+
+**Note:** Google does not make the play store data available for export every day. The export job checks for new files every day. However, having monitored the job, it appears the data is made available every 7 - 14 days, and seems to primarily be made available on weekends. Due to this lack of consistency, there will be delays in the data available for this dataset. The data currently in BigQuery is the most current data available from Google.
+
+## Accessing the Data
+
+Access the data at moz-fx-data-marketing-prod.google_play_store
+
+# Data Reference
+
+## Schema
```

Comment:
Does this schema cover all tables or just one of them? If the latter, maybe we can just leave this section out. This tends to go out of date super quickly anyway...

Reply:
Covers all the tables. The dimension is the only thing that changes but metrics stay the same for the retained_installer tables.